### PR TITLE
⭐️ update policies to v2

### PR DESCRIPTION
**Pull request by imilchev**

### CODE REVIEW

File: community/chef-infra-server.mql.yaml [hunk 1]
```diff
@@ -1,44 +1,38 @@
...
-          eol-reporting-addon:
-          eol-push-jobs-addon:
-          eol-analytics-addon:
-          eol-ha-addon:
-          pivotal-pem-permissions:
-          webui-pem-permissions:
-          secure-tls-only:
-          secrets-file-permissions:
-          disable-insecure-addon-compat:
+      desc: "Chef Infra Server Policy identifies several misconfigurations and end of life components that allow attackers to expose node information:\n  - Insecure disk permissions on critical directories and configuration files.\n  - End of life components installed on the Chef Infra Server such as Push Jobs, Analytics, or Reporting, which no longer receive security updates.\n  - Insecure servers settings such non-secure TLS support or legacy add-on compatibility.\n  \nIf you have questions, comments, or have identified ways to improve this policy, please write me at tim@mondoo.com, or reach out in the [Mondoo Slack Community](https://mondoo.link/slack)."
```
Comment (tas50) [inline community/chef-infra-server.mql.yaml:13]: @chris-rock Any way we could get the automatic formatting to use `|-` when there are line breaks in the text? These become super hard to hand edit now.

Comment (chris-rock) [reply community/chef-infra-server.mql.yaml:13]: Yeah, unclear why this happens because it does not happen all the time.

**Comment by chris-rock** [comment]
@imilchev can rebuild the PR on the latest changes?

**Review by chris-rock — APPROVED** [review]
Thank you @imilchev

**Pull request merged**